configure.ac

```diff
@@ -9,6 +9,14 @@ AC_TYPE_SSIZE_T
 AC_FUNC_MALLOC
 AC_FUNC_REALLOC
 AC_CHECK_FUNCS([memmove setlocale strdup])
+AC_SEARCH_LIBS([strlcpy], [bsd], [], [
+  AC_MSG_ERROR([unable to find strlcpy() function])
+])
```

Comment:
Yes it will error out,  I guess the message should be "unable to find strlcpy(). Make sure libbsd is installed"  

Guess if #10 gets merged you can remove the whole `AC_SEARCH_LIBS`  since you are not dependent on `libbsd`  after #10. 

This was more  to solve the current issue of warning the user that there was no `strlcpy` at `./configure`  instead of at compile time. It depends on when are you going to merge #10, you can always remove this `AC_SEARCH_LIBS` after merging #10.